### PR TITLE
Adding BillingProfile Status, Reason, SpendingLimit and accountType.

### PR DESCRIPTION
**Pull request by asarkar84**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Review by yungezz — APPROVED** [review]

File: specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json [hunk 1]
```diff
@@ -5352,7 +5352,7 @@
             "modelAsString": true
           }
         },
-        "customerType": {
+        "accountType": {
```
Comment (majastrz) [inline specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json]: >accountType [](start = 9, length = 11)

This is a breaking changes. Can you introduce this in a new API version?

**Review by majastrz — CHANGES_REQUESTED** [review]
The change introduces a breaking change. Please take a look at my comment.

File: specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json (hunk 1 above)
Comment (asarkar84) [reply specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json]: Added customerType back. accountType  will be a new property. Will remove customerType in next version.

**Comment by asarkar84** [comment]
@majastrz Please take a look. I have added the customerType  property back. Will remove it in next version.

File: specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json (hunk 1 above)
Comment (majastrz) [reply specification/billing/resource-manager/Microsoft.Billing/preview/2019-10-01-preview/billing.json]: Makes sense. Thanks for fixing...

---
In reply to: [370401325](https://github.com/Azure/azure-rest-api-specs/pull/8202#discussion_r370401325) [](ancestors = 370401325)

**Review by majastrz — APPROVED** [review]
Signed off from ARM side.

**Comment by asarkar84** [comment]
@yungezz Please merge the PR.

**Pull request merged**